docs/maps/index.asciidoc

```diff
@@ -65,4 +65,5 @@ include::map-settings.asciidoc[]
 include::connect-to-ems.asciidoc[]
 include::import-geospatial-data.asciidoc[]
 include::indexing-geojson-data-tutorial.asciidoc[]
+include::clean-data.asciidoc[]
```

Comment:
Lets put clean-data.asciidoc before indexing-geojson-data-tutorial.asciidoc. I think it provides more vital information then the tutorial